connegp-implementation-report/config.js

```diff
@@ -56,11 +56,12 @@ var respecConfig = {
         "PROF-IETF": {
             authors: [
                 "L. Svensson",
-                "R. Verborgh"
+                "R. Verborgh",
+                "H. Van de Sompel"
             ],
-            href: "https://profilenegotiation.github.io/I-D-Profile-Negotiation/I-D-Profile-Negotiation",
-            title: "Indicating and Negotiating Profiles in HTTP",
-            date: " 2019-07-11",
+            href: "https://datatracker.ietf.org/doc/draft-svensson-profiled-representations/01/",
```

Comment:
I'd say it's better to point to the github version of the I-D until we've clarified with IETF if we can have it accepted by an IETF WG. The github version is the most recent one.